alexandria/core/management/commands/encrypt_files.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from tqdm import tqdm
+
+from alexandria.core.models import File
+
+
+class Command(BaseCommand):
+    help = "Swaps plain text file content to encrypted content"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--dry", dest="dry", action="store_true", default=False)
+
+    def handle(self, *args, **options):
```

Comment:
This needs some settings validation. If you run this command without encryption properly setup, it will just re-upload the unencrypted files, but set the encryption method to whatever's in the settings.

We should be sure that encryption is enabled and configured correctly before commencing with this task

Reply:
Done
https://github.com/projectcaluma/alexandria/blob/e941029fae378f57e5ace86171472b96b5c89956/alexandria/core/management/commands/encrypt_files.py#L15-L24

Will be adding a test for the command, not sure coverage doesnt complain